### PR TITLE
[INLONG-3173][Sort-Standalone] Unify SortClusterConfig in manager and sort-standalone

### LINKED ISSUE #3173: [Bug][Sort-Standalone] Unify SortClusterConfig in manager and sort-standalone
### Description

parent issue #3115

unify the sturcture 
and
remove "sinkType" from sink params, which cause initilization failure in Sort-Standalone

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Title Name: [INLONG-3173][Sort-Standalone] Unify SortClusterConfig in manager and sort-standalone

Fixes #3173 

unify the sturcture
and
remove "sinkType" from sink params, which cause initilization failure in Sort-Standalone

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by healchow — APPROVED** [review]

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sort/SortClusterConfigResponse.java [hunk 1]
```diff
@@ -43,8 +43,8 @@
     @Builder
     @Data
     public static class SortTaskConfig {
-        String taskName;
-        SinkType sinkType;
+        String name;
+        SinkType type;
```
Comment (luchunliang) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sort/SortClusterConfigResponse.java:47]: It is better that type is String.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sort/SortClusterConfigResponse.java [hunk 2]
```diff
@@ -31,7 +31,7 @@
     String msg;
     int code;
     String md5;
-    List<SortTaskConfig> tasks;
+    SortClusterConfig data;
```
Comment (luchunliang) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sort/SortClusterConfigResponse.java:34]: pojo class can move to inlong-common module.
And remove pojo class of sort-standalone module "org.apache.inlong.sort.standalone.config.pojo".
Use same class in sort-standalone and manager.

**Review by wardlican — APPROVED** [review]

**Review by luchunliang — APPROVED** [review]

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sort/SortClusterConfigResponse.java (hunk 1 above)
Comment (vernedeng) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sort/SortClusterConfigResponse.java:47]: fix in next pr

File: inlong-common/src/main/java/org/apache/inlong/common/pojo/sortstandalone/SortTaskConfig.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+package org.apache.inlong.common.pojo.sortstandalone;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+import java.util.Map;
+
+@Data
+@Builder
```
Comment (healchow) [inline inlong-common/src/main/java/org/apache/inlong/common/pojo/sortstandalone/SortTaskConfig.java:27]: Please add `NoArgsConstructor` and `AllArgsConstructor` annotations for the pojo classes.

Comment (vernedeng) [reply inlong-common/src/main/java/org/apache/inlong/common/pojo/sortstandalone/SortTaskConfig.java:29]: thx, fixed

**Review by healchow — APPROVED** [review]

**Pull request merged**